server/api/controllers/ArticleController.js

```diff
@@ -0,0 +1,13 @@
+import models from '../../models';
+
+export const createArticle = async (req, res) => {
+  try {
+    const newArticleData = req.body;
+    const userId = req.user.id;
+    newArticleData.authorId = userId;
+    const newArticle = await models.Article.create(newArticleData);
+    res.status(201).send({ data: newArticle });
+  } catch (error) {
+    res.status(502).send({ message: 'OOPS! an error occurred while trying to create your article, you do not seem to be logged in or signed up, log in and try again!' });
```

Comment:
@Princesso Please add a `return` statement to line `11`, and change the `status` code to `500`.

Reply:
Change effected.